universal-login-ops/src/dev/startDevelopmentRelayer.js

```diff
@@ -1,7 +1,12 @@
-import {DevelopmentRelayer} from 'universal-login-relayer';
+import Relayer, {DevelopmentRelayer} from 'universal-login-relayer';
 
-async function startDevelopmentRelayer(configuration, wallet) {
-  const relayer = new DevelopmentRelayer(configuration, wallet.provider);
+async function startDevelopmentRelayer(configuration, wallet, basicRelayer) {
```

Comment:
Instead of passing `basicRelayer : boolean`, pass relayer class/constructor.
You can also create a factory function that based on a string will provide new instance and use it to add option to CLI.